Event/ConsoleMonitoringEvent.php

```diff
@@ -37,9 +37,6 @@ protected static function getUnderscoredEventCommandName(ConsoleEvent $event): ?
 
     protected static function getPeakMemory()
     {
-        $memory = memory_get_peak_usage(true);
-        $memory = ($memory > 1024 ? intval($memory / 1024) : 0);
-
-        return $memory;
+        return memory_get_peak_usage(true);
```

Comment:
Units management is hard… what about be more explicit about units? `getPeakMemoryInBytes`?
We can also add units after magic constants `$time / 1000 /*secondes*/`

Reply:
Will do !

Reply:
Will do for the memory.
For the timer, it could be anything as long as your value is unchanged between you code, and when outputed on your graphs.